src/platforms/common/enriching-events/breadcrumbs.mdx

```diff
@@ -53,7 +53,7 @@ You can choose from the following breadcrumb log levels: `"fatal"`, `"critical"`
 
 <PlatformContent includePath="enriching-events/breadcrumbs/automatic-breadcrumbs" />
 
-<PlatformSection notSupported={["unreal"]}>
+<PlatformSection notSupported={["unreal", "rust", "elixir"]}>
```

Comment:
also supported in rust: https://docs.rs/sentry/latest/sentry/struct.ClientOptions.html#structfield.before_breadcrumb